Pa3/Durchführung/Project/app/State/InitializeMcu.c

```diff
@@ -36,11 +40,28 @@
 
 EventEnum InitializeMcu_InitializeAll(void)
 {
-
+  //HAL
```

Comment:
please use the comment style provided in the Style Guide. Also could add some flavour
```suggestion
  /* Initialize HAL modules */
```